website/server/libs/bannedWords.js

```diff
@@ -0,0 +1,3 @@
+let bannedWords = ['TEST_PLACEHOLDER_SWEAR_WORD_HERE'];
+
+module.exports = bannedWords;
```

Comment:
We can reenable the ability to direct commit for this one thing. I very much don't want the words in this PR because coders will have to look through the PR to see what's been done and avoiding the list of bad words then will be difficult. The comment won't help much in that case because they can't stop scrolling through the PR at that point. It's only useful when the file is the only thing being viewed.

I know this is an extreme approach but we make a big effort to not expose Habiticans to bad words, and I'd like us to do this to include our contributor developers in the people we protect.